js-variables-and-numbers/tip-calculator/scripts.js

```diff
@@ -0,0 +1,7 @@
+const mealCost = 100;
+const tipPercentage = 0.25;

```

Comment:
here I did "let tipPercentage", because perhaps this value will change from customer to customer?

Reply:
"let" we put if the variable changes lower in the code, as in the first challenge. Here we don't change it, but only refer to the value. It is logical to assume that these values will be changed by the user at some point, for example, the user will enter a form field, but this is not specified in the task.